embedded/build.gradle

```diff
@@ -26,8 +29,186 @@ dependencies {
 
   compile project(path: ':common', configuration: 'api')
   compile project(path: ':common', configuration: 'spi')
+
+  metadata rootProject.libraries.saxon
+  metadata rootProject.libraries.saxon_dom
+
+  metadata rootProject.libraries.shrinkwrap_descriptors_ant
+  metadata rootProject.libraries.shrinkwrap_descriptors_metadata_parser
+
+  compile files(sourceSets.main.output.classesDir)
+}
+
+ext.metadataLibs = project.file("tmp/libs").toString()
+ext.metadataSrcDir = project.file("tmp/src/main").toString()
+
+task compileMetadata(type: JavaCompile){
+  source = metadataSrcDir
+  classpath = compileJava.classpath + configurations.metadata
+  destinationDir =  sourceSets.main.output.classesDir
+  doLast {
+    copy {
+      from file("${metadataSrcDir}/resources/META-INF/services")
+      into file("${buildDir}/resources/main/META-INF/services")
+    }
+    project.file("tmp").deleteDir()
+  }
+}
+
+task generateJCAMetadata() {
+  copy {
+    from configurations.metadata.files
+    into metadataLibs
+  }
+
+  ant.path(id: 'metadata') {
+    fileset(dir: metadataLibs)
+  }
+
+  ant.taskdef(name: 'metadataParser',
+    classname: 'org.jboss.shrinkwrap.descriptor.extension.ant.task.MetadataParserTask',
+    classpathRef: 'metadata')
+
+  ant.metadataParser(verbose:"false", classpathRef: 'metadata') {
+    path(pathToApi: "${metadataSrcDir}/java",
+         pathToImpl: "${metadataSrcDir}/java",
+         pathToServices: "${metadataSrcDir}/resources/META-INF/services")
+    descriptors() {
+      descriptor(pathToXsd: project(':common').file("src/main/resources/dtd/connector_1_0.dtd"),
+                 nameSpace: "j2ee",
+                 packageApi: "org.ironjacamar.shrinkwrap.descriptor.api.connector10",
+                 packageImpl:"org.ironjacamar.shrinkwrap.descriptor.impl.connector10",
```

Comment:
These need to stay at "org.jboss.shrinkwrap" in order to align with the official JARs